File: ruby/tests/503-load_info-bourreau.rb
admin   = admin_handle

raise "SKIPPED: no Bourreaux" unless @bourreau_list_500.present?

bourreau_id   = @bourreau_list_500.first[:id]
bourreau_load_info = admin.load_info_bourreau({:bourreau_id => bourreau_id})

puts "BOURREAU_LOAD_INFO: #{bourreau_load_info.inspect}" if @debug

raise "Value returned is not an hash" unless bourreau_load_info.present? && bourreau_load_info.is_a?(Hash)
raise "You have no information for your bourreau" if bourreau_load_info.empty?

true

File: perl/tests/503-load_info-bourreau.pl
my $admin      = &AdminHandle;
our $bourreau_list_500;

die "SKIPPED: no Bourreaux\n" if scalar(@$bourreau_list_500) == 0;

my $bourreau_id   = $bourreau_list_500->[0]->{'id'};
my $bourreau_load_info = $admin->load_info_bourreau({"bourreau_id" => $bourreau_id});

if ($DEBUG) {
  my $dump_bourreau_load_info = Data::Dumper::DumperX($bourreau_load_info);
  print "BOURREAU_LOAD_INFO: $dump_bourreau_load_info\n";
}

die "Value returned not an hash.\n" unless defined($bourreau_load_info) && ref($bourreau_load_info) eq "HASH";

1;
